### PR TITLE
Consume tf2pulumi from pulumi-terraform-bridge.

### PR DESCRIPTION
**Pull request by pgavlin**
And delete the implementation here.

### CODE REVIEW

**Comment by komalali** [comment]
@lukehoban @stack72 we need to get this PR in because tf2pulumi is super out of date.

**Comment by pgavlin** [comment]
@komalali thanks for bringing this PR up to date. @lukehoban @stack72 @justinvp @lblackstone can one of y'all take a look at this?

**Comment by lblackstone** [comment]
It looks like there's a [test failure](https://github.com/pulumi/tf2pulumi/pull/207/checks?check_run_id=1787812632#step:20:565): `Exception: 'depends_on' was passed a value that was not a Resource.`

**Comment by pgavlin** [comment]
Yep, looking into that now.

**Comment by pgavlin** [comment]
Alright, tests are green

File: tests/terraform/aws/aws_test.go [hunk 1]
```diff
@@ -64,7 +64,8 @@ func TestEIP(t *testing.T) {
 }
 
 func TestELB(t *testing.T) {
-	RunAWSTest(t, "elb")
+	// Skip Python due to a bug in depends_on codegen
```
Comment (lblackstone) [inline tests/terraform/aws/aws_test.go:67]: Is this tracked somwhere?

**Review by lblackstone — APPROVED** [review]

File: tests/terraform/aws/aws_test.go (hunk 1 above)
Comment (pgavlin) [reply tests/terraform/aws/aws_test.go:67]: https://github.com/pulumi/pulumi/issues/6220

**Pull request merged**